nomenclature/codelist.py

```diff
@@ -52,6 +52,36 @@ def keys(self):
     def values(self):
         return self.mapping.values()
 
+    @classmethod
+    def from_list(
```

Comment:
Since this method is used only once I would prefer to remove it and just include the code in `DataStructureDefinition.__init__`. Unless you already have some more use cases in mind for this function, then I would say it's fine to keep.

Reply:
Done